portfolio/src/main/webapp/script.js

```diff
@@ -116,12 +116,15 @@ function createCommentElement(comment) {
 
 function commentLogin() {
   fetch('/login').then(response => response.json()).then((status) => {
-    if (status) {
+    console.log(status);
```

Comment:
If this is for debugging purpose, let's remove it before merging

Reply:
Oops! Will delete.